src/parser/set/set.factory.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (split, SET) => {
+    return command => {
+        const withoutPrefix = command.slice('set url for '.length);
```

Comment:
so we don't need a user in a team to specify their own team but we should inject this information into the action. This is because when we get a slash command request, Slack indicates the team that it's coming from